### PR TITLE
PR for BXMSDOC-7825: Release notes docs for DM and PAM 7.12

### PR DESCRIPTION
**Pull request by kaldesai**
**JIRAs:** 

- [Epic](https://issues.redhat.com/browse/BXMSDOC-7825)

**Doc previews:**

- [Release notes for Red Hat Process Automation Manager 7.12](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7825-7.12-RHPAM-RN/)
- [Release notes for Red Hat Decision Manager 7.12](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7825-7.12-RHDM-RN/)

### CODE REVIEW

File: doc-content/enterprise-only/release-notes/rn-7.12-fixed-issues-ref.adoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+* When you set the `org.jbpm.wb.forms.renderer.ext` system property to `true` and you try to attach a document, the *Upload* button fails to display [https://issues.redhat.com/browse/RHPAM-3515[RHPAM-3515]]
+* When you clone a Git project from outside of {CENTRAL}, you can clone the project even though it is not shown in {CENTRAL} because of lack of permissions [https://issues.redhat.com/browse/RHPAM-3375[RHPAM-3375]]
+* When you migrate a Git project from version 6.4 to version 7.8.1, you receive an error message [https://issues.redhat.com/browse/RHPAM-3232[RHPAM-3232]]
+* Editing a data object in the guided decision table fails with the null response [https://issues.redhat.com/browse/RHDM-1781[RHDM-1781]]
+* The test scenario tool is unable to execute models using imported inputs and/or decisions nodes and you receive an error message [https://issues.redhat.com/browse/RHDM-1645[RHDM-1645]]
+
+== Build and assembly
+
+* When a system logs contains an attacker controlled string value, remote code execution happens in the Java logging library of Log4j 2.x [https://issues.redhat.com/browse/RHPAM-4077[RHPAM-4077]]
```
Comment (emmurphy1) [inline doc-content/enterprise-only/release-notes/rn-7.12-fixed-issues-ref.adoc:21]: RHPAM-4077 not available outside of the firewall. Contact SME to make available or remove.

File: doc-content/enterprise-only/release-notes/rn-kogito-productize-features-ref.adoc [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+This section highlights the {KOGITO} features included with {PRODUCT}.
 
-* Layering on https://access.redhat.com/products/quarkus[{QUARKUS}] or https://access.redhat.com/products/spring-boot[Spring Boot] to enable developers to run {KOGITO} microservices on their preferred cloud-native application development stack
-* Automated decision-making capabilities based on rule units to run within defined business constraints
-* Example applications containing various types of {KOGITO} decision microservices on https://access.redhat.com/products/quarkus[{QUARKUS}] or https://access.redhat.com/products/spring-boot[Spring Boot] to help you develop your applications
+== Support for BPMN processes in {KOGITO}
 
-For more information about getting started with {KOGITO} in {PRODUCT}, see {URL_GETTING_STARTED_KOGITO_MICROSERVICES}#assembly-getting-started-kogito-microservices[_{GETTING_STARTED_KOGITO_MICROSERVICES}_].
+{PRODUCT} now provides support for straight-through processes in {KOGITO} to enable you to develop process microservices using Business Process Model and Notation (BPMN) 2.0 models. The supported process services include:
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-kogito-productize-features-ref.adoc:10]: Done

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 3]
```diff
@@ -3,222 +3,177 @@
 
 This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-== {CENTRAL}
+== Ability to send cookies in REST workItemHandler
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:6]: I verified with the dev and it is "the REST".

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 4]
```diff
@@ -3,222 +3,177 @@
...
-=== Deployment of {PRODUCT} Spring Boot business applications on {OPENSHIFT}
+When you deploy {PRODUCT} on {OPENSHIFT} using the operator installer, by default the deployment uses the `OpenShiftStartupStrategy` setting. You can now, if it is necessary, switch to the Controller startup strategy in the configuration user interface.
 
-Documentation about deploying {PRODUCT} Spring Boot business applications on {OPENSHIFT} is now provided. For more information see, {URL_INTEGRATING}#openshift-springboot-proc_business-applications[_{CREATING_SPRING_BOOT_APPLICATIONS}_].
+=== Improved SSL configurability
+You can now enable or disable SSL in the operator, and expose the SSL route.
 
-=== Integration with Spring Boot version 2.3.4
+=== Default configuration for CORS
+You can now specify whether to use a default or user customized CORS configuration.
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:143]: Spelt out CORS in the content

File: doc-content/enterprise-only/release-notes/rn-7.12-known-issues-ref.adoc [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
+
+== {CENTRAL}
+
+.In the test scenarios table, `today()` and `now()` functions are not evaluated correctly [https://issues.redhat.com/browse/RHDM-1816[RHDM-1816]]
+
+Issue: In the test scenarios table, `today()` and `now()` functions are not evaluated correctly. For example, the `string(today())` function should produce output similar to `2021-05-11` but instead it displays a null value.
+
+Steps to reproduce:
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-7.12-known-issues-ref.adoc:12]: Okay, noted! Thank you!

File: doc-content/enterprise-only/release-notes/rn-7.12-fixed-issues-ref.adoc (hunk 1 above)
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-7.12-fixed-issues-ref.adoc:21]: Yes, I have contacted SMEs. They are going to make it public.

**Pull request merged**